.github/actions/spelling/allow/README.md

```diff
@@ -1,4 +1,4 @@
-# Dictionaries are lists of words to accept unconditionally
+# Allow files are lists of words to accept unconditionally
```

Comment:
As noted, the dictionary directory is being changed to just a list of supplements to the base dictionary.